File: eNote/UI/enote_desktop/lib/models/enums/stanje_upisa.dart
// ignore_for_file: constant_identifier_names

import 'package:json_annotation/json_annotation.dart';

part 'stanje_upisa.g.dart';

@JsonEnum(alwaysCreate: true)
enum StanjeUpisa { NaCekanju, Potvrdjeno, Aktivno, Otkazano, Odustao }

File: eNote/eNote.Model/Enums/StanjeUpisa.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Text.Json.Serialization;

namespace eNote.Model.Enums
{
    public enum StanjeUpisa
    {
        NaCekanju = 1,
        Potvrdjeno = 2,
        Aktivno = 3,
        Odbijeno = 4,
        Odustao = 5
    }
}
